### PR TITLE
User Validation

### PR DESCRIPTION
**Pull request by cho102**

### CODE REVIEW

**Review by Arnav-Menon — APPROVED** [review]
LGTM

**Comment by Arnav-Menon** [comment]
also, just a heads up, don't forget to attach screenshots and maybe a tiny description of what you did

**Comment by cho102** [comment]
created new branch

**Review by kninja99 — COMMENTED** [review]
Do you think you can attach a img of what this currently looks like, thanks

File: components/Login.jsx [hunk 1]
```diff
@@ -1,31 +1,78 @@
...
+
+  useEffect(() => {
+    if (Object.keys(formErrors.length === 0) && isSubmit) {
+      console.log(formInput);
+    }
+  });
+
+  const validate = (values) => {
+    const errors = {};
+    const regex = /^[a-zA-Z]{3,4}[0-9]{3}@ucr\.edu$/i;
```
Comment (Arnav-Menon) [inline components/Login.jsx:33]: nothing wrong here, just curious. how does regex work? like what is this statement saying?

**Review by Arnav-Menon — APPROVED** [review]

**Review by shahdivyank — APPROVED** [review]

**Pull request merged**